src/lib/core/overlay/position/position-strategy.ts

```diff
@@ -3,4 +3,7 @@ export interface PositionStrategy {
 
   /** Updates the position of the overlay element. */
   apply(element: Element): Promise<void>;
+
+  /** Cleans up any DOM modifications made by the position strategy, if necessary. */
+  dispose?(): void;
```

Comment:
Let's make this required and give `ConnectedPositionStrategy` a no-op `dispose`, then we won't have to check whether it exists before calling it.